spec/controllers/mailings_controller_spec.rb

```diff
@@ -6,17 +6,14 @@
   let(:mailing)        { FactoryGirl.create(:mailing) }
   let(:user)           { FactoryGirl.create(:user) }
 
-  shared_context 'User logged in' do
-    before { sign_in user }
-  end
-
   describe 'GET index' do
     before { mailing }
```

Comment:
It should be a `let!`, yes. But it's been there before so technically not part of this PR ;)